Verification/SPIDVerificationAPI_WPF_Sample/EnrollPage.xaml.cs

```diff
@@ -96,8 +105,17 @@ public EnrollPage()
             {
                 setStatus("Using profile Id: " + _speakerId.ToString());
                 refreshPhrases();
-                string enrollmentsStatus = _storageHelper.readValue(MainWindow.SPEAKER_ENROLLMENTS);
-                if ((enrollmentsStatus != null) && (enrollmentsStatus.Equals("Done")))
+
+                Profile profile = await _serviceClient.GetProfileAsync(_speakerId);
```

Comment:
There is no need to call GetProfile here. You can read the value from the stored file.